charters/UploadService/charter.md

```diff
@@ -0,0 +1,78 @@
+# Upload Service
+
+## Description
+The Upload Service enables fast, cost-effective upload, staging, and validation of data submitted to the Human Cell Atlas (HCA) Data Coordination Platform (DCP) from across the world.
```

Comment:
I'm not sure 'fast, cost-effective' adds anything here. Fast and cost-effective compared to what?

What's the difference between upload and staging? Using upload consistently throughout would seem to make sense

Reply:
'fast, cost-effective' is an important focus point here. SamP and Parth spent significant time configuring our AWS s3 setup to ensure it would be fast and not costly.

Upload is sending data into s3. Staging is holding it until DSS is ready to accept it.